index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>To-Dos App</title>
+</head>
+  <body>
+      <h1>To-Dos</h1>
+      <form action="#">
+        <input type="text" id="taskInput" placeholder="Enter a task">
+        <input type="submit" value="+">
+      </form>
+      <br>
```

Comment:
@jjw18th Using a br tag here is considered bad practice. Please look into why that is the case and fix this accordingly.

Reply:
This is being tracked by #2 . Resolving.